diff --git a/google/genai/tests/types/test_types.py b/google/genai/tests/types/test_types.py
index 531a5c5..a498902 100644
--- a/google/genai/tests/types/test_types.py
+++ b/google/genai/tests/types/test_types.py
@@ -18,7 +18,8 @@ import copy
 import json
 import sys
 import typing
-from typing import Optional, assert_never
+from typing import Optional
+from typing_extensions import assert_never
 import PIL.Image
 import pydantic
 import pytest
